WatchParty/backend/test/movie_model.test.js

```diff
@@ -1,22 +1,19 @@
-const chai = require('chai');
-chai.use(require('sinon-chai'));
```

Comment:
To test the model associations, I used the `calledWith` method that is provided by `sinon-chai`. For some strange reason, eliminating this line in all test files it complains with a missing function in all tests. However, removing the line from one test file allows all the tests to work, which makes no sense to me. I recommend leaving it in all test files. It might be a zany situation, but at least our code is consistent.

Reply:
I remove all the tests so far.